core/config.js

```
@@ -0,0 +1,31 @@
+// *************** IMPORT LIBRARY ***************
+require('dotenv').config();
+const { ApolloError } = require('apollo-server');
+
+/**
+ * Validates and provides access to environment configuration variables
+ * Ensures all required variables are present before application start
+ * 
+ * @returns {Object} - Object containing validated configuration values
+ * @throws {ApolloError} - If required environment variables are missing
+ */
+function GetConfig() {
+  // *************** Database Configuration ***************
+  const mongodbUri = process.env.MONGODB_URI;
```

fail fast + clear error!